bin/apisix

```diff
@@ -530,6 +531,10 @@ local function init()
         sys_conf["worker_processes"] = "auto"
     end
 
+    if sys_conf["enable_reuseport"] == false then
+        enable_reuseport = false
+    end
+
```

Comment:
we can use the `enable_reuseport` in `config.yaml`

Reply:
done